### PR TITLE
Perks & benefits changes

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
1.Change title font size to 1.5rem
2.Replace content "we are hiring" with "5 days working" and change description.
3.Change order of perks-benfits section.
4.Remove style tag.

### CODE REVIEW

File: vue-frontend/src/components/jobs/PerksAndBenifits.vue [hunk 1]
```diff
@@ -95,13 +98,12 @@ export default {
         },
         {
           id: 2,
-          title: "Learning & Development",
-          icon: learning,
-          alt: "learning-icon",
+          title: "5 Days Work Week",
+          icon: hiring,
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/PerksAndBenifits.vue:102]: See Figma design for icon

File: vue-frontend/src/components/jobs/PerksAndBenifits.vue [hunk 2]
```diff
@@ -49,7 +49,10 @@
               class="normal-2-text tw-flex-none title tw-inline-flex tw-items-center tw-text-black-900 tw-font-bold"
             >
               <img :src="perk.icon" loading="lazy" :alt="perk.alt" />
-              <span class="tw-ml-[10px]">{{ perk.title }}</span>
+              <span
+                class="tw-ml-[10px] tw-text-[1.5rem] md:tw-text-[1.1rem] lg:tw-text-[1.5rem]"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/jobs/PerksAndBenifits.vue:53]: Why do we need 1.1rem in md?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/jobs/PerksAndBenifits.vue (hunk 2 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/jobs/PerksAndBenifits.vue:53]: it was looking messy in md so need to resize it.

Comment (cp-sumi-k) [reply vue-frontend/src/components/jobs/PerksAndBenifits.vue:53]: Fix description in > 768 width devices

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**